gems/pending/appliance_console.rb

```diff
@@ -289,11 +290,14 @@ module ApplianceConsole
 
           if agree("Apply time and timezone configuration? (Y/N): ")
             say("Applying time and timezone configuration...")
-            Env['TIMEZONE'] = new_loc, new_city
-            Env['TIME'] = new_date, new_time unless Env.error?
+            res = AwesomeSpawn.run("timedatectl set-timezone \"#{new_loc}/#{new_city}\"")
+            if res.exit_status == 0
+              res = AwesomeSpawn.run("timedatectl set-time \"#{new_date} #{new_time}\"",
+                                     :params => {"adjust-system-clock".to_sym => nil})
+            end
```

Comment:
Ah nice, I'm a bit unclear about how AwesomeSpawn deals with some of the command line parameters, wouldn't it need to be:

``` ruby
AwesomeSpawn.run!("timedatectl", :params => {nil => ["set-timezone", "#{new_loc}/#{new_city}"]})
```
based on https://github.com/ManageIQ/awesome_spawn/blob/master/lib/awesome_spawn/command_line_builder.rb#L40
or

``` ruby
AwesomeSpawn.run!("timedatectl", :params => [["set-timezone", "#{new_loc}/#{new_city}"]])
```
Based on https://github.com/ManageIQ/awesome_spawn/blob/master/lib/awesome_spawn/command_line_builder.rb#L43, or are you using this form and that outer array is just unnecessary?

And I escaped the quotes because I wasn't sure how whatever underlying mechanism AwesomeSpawn uses would deal with parameters with spaces.  Is it safe to leave those values unquoted if they might have spaces in them?